src/gui/systray.h

```diff
@@ -53,13 +54,23 @@ class Systray
     void showMessage(const QString &title, const QString &message, MessageIcon icon = Information);
     void setToolTip(const QString &tip);
     bool isOpen();
+    Qt::WindowFlags windowFlags() const;
+    qreal windowRadius() const;
+    qreal windowWidth() const;
+    qreal windowHeight() const;
 
     Q_INVOKABLE void pauseResumeSync();
     Q_INVOKABLE bool syncIsPaused();
     Q_INVOKABLE void setOpened();
     Q_INVOKABLE void setClosed();
     Q_INVOKABLE void positionWindow(QQuickWindow *window) const;
     Q_INVOKABLE void forceWindowInit(QQuickWindow *window) const;
+    Q_INVOKABLE void onActiveChanged(QQuickWindow *window);
+
+    Q_PROPERTY(Qt::WindowFlags windowFlags READ windowFlags)
```

Comment:
please put the Q_PROPERTY just behind the Q_OBJECT macro
having them inline inside the class is very unusual